CMakeLists.txt

```diff
@@ -428,4 +428,4 @@ if(GINKGO_CONFIG_LOG_DETAILED)
 else()
     FILE(READ ${PROJECT_BINARY_DIR}/minimal.log GINKGO_LOG_SUMMARY)
 endif()
-MESSAGE("${GINKGO_LOG_SUMMARY}")
+MESSAGE(STATUS "${GINKGO_LOG_SUMMARY}")
```

Comment:
just checking: is this the right thing? it adds -- to the summary line, was it missing?

Reply:
It does this in one line, otherwise, it's not too bad. The problem is, without this the message is printed to `stderr`, and the custom runner interprets that as an error. In general, I think we should minimize the amount of `stderr` traffic for non-errors.

See (none) or NOTICE here: https://cmake.org/cmake/help/latest/command/message.html